src/test/java/bio/terra/stairway/flights/TestFlightRetry.java

```diff
@@ -29,11 +30,15 @@ public TestFlightRetry(FlightMap inputParameters, Object applicationContext) {
       retryRule =
           new RetryRuleExponentialBackoff(
               initialIntervalSeconds, maxIntervalSeconds, maxOperationTimeSeconds);
+    } else if (StringUtils.equals("random", retryType)) {
```

Comment:
Can we use an enum with string constant metadata for the retry type? E.g. `RETRY_TYPE.random.name().equals(retryType)`

Reply:
It is testing code with one caller. I don't think it is worth it.